api/types.go

```diff
@@ -608,6 +608,19 @@ func FormatParams(params map[string][]string) (map[string]interface{}, error) {
 		} else {
 			field := valueOpts.FieldByName(opt.Name)
 			if field.IsValid() && field.CanSet() {
+				if reflect.PointerTo(field.Type()) == reflect.TypeOf((*TriState)(nil)) {
```

Comment:
could this fit into the `field.Kind() {` below?

Reply:
Not without restructuring the way the switch works - field.Kind returns int, not TriState.